client/did.go

```diff
@@ -150,24 +160,33 @@ func (c *Client) CreateDID(cfg *did.DIDCreate) (string, bool) {
 }
 
 func (c *Client) SetupDID(dc *did.DIDCreate) (string, bool) {
-	if dc.Type < did.BasicDIDMode && dc.Type > did.WalletDIDMode {
+	if dc.Type < did.LightDIDMode && dc.Type > did.WalletDIDMode {
 		return "Invalid DID mode", false
 	}
-	if !strings.Contains(dc.PubImgFile, did.PubShareFileName) ||
-		!strings.Contains(dc.DIDImgFileName, did.DIDImgFileName) ||
-		!strings.Contains(dc.PubKeyFile, did.PubKeyFileName) ||
-		!strings.Contains(dc.QuorumPubKeyFile, did.QuorumPubKeyFileName) ||
-		!strings.Contains(dc.QuorumPrivKeyFile, did.QuorumPvtKeyFileName) {
-		return "Required files are missing", false
-	}
+
 	switch dc.Type {
+	case did.LightDIDMode:
```

Comment:
It'll be useful if a short description on each mode is added in the readme.

Reply:
Since ReadMe only contains commands, so I have not added any description to ReadMe. But I have added descriptions to the functions in the code.